### PR TITLE
Split "Setting up" in two topics

### PR DESCRIPTION
**Pull request by iglpdc**
"Setting up" contains two different parts: first, a motivation and basics of a VCS, and, second, instructions to set up Git locally. This splits the file in two.

It also fixes a warning about having subheading on the original file.

Feedback on whether we want to make the split or not are welcome.

**Don't merge yet** as this is dependent on #116.

### CODE REVIEW

File: 02-setup.md [hunk 1]
```diff
@@ -9,38 +9,6 @@ minutes: 10
 > *   Explain which initialization and configuration steps are required once per computer,
```
Comment (iglpdc) [inline 02-setup.md]: Why are we saying "configuration per computer", and "per repository". We only explain here the steps needed to configure Git "per computer". If this is refer to the `--global` file, I think the learning objectives can be simplified to:

- Configure `git` the first time is used on a computer
- Understand the meaning of the `--global` configuration flag

**Comment by BernhardKonrad** [comment]
This is great! +1 for merging.

**Comment by msarahan** [comment]
+1 looks good to me.

**Comment by PBarmby** [comment]
+1 as well.

**Comment by iglpdc** [comment]
Thanks @BernhardKonrad and @msarahan. I'm merging then.

**Comment by swaldman3** [comment]
Agree with the split - especially useful for people after a course who want a quick reminder of the commands to configure git but don't need to go through the motivations again.

**Pull request merged**